website/public/404.html

```diff
@@ -0,0 +1,128 @@
+<!doctype html>
```

Comment:
I think we should consider removing the build generated files under `public/` as well as those built under `../docs/` since they cause a ton of noise in these PRs. Thoughts?

Even if we decide to keep these built files in the PR, there's no reason we'd need to have both these in `public/` and those in `../docs/`

Reply:
i have removed `website/public`

we will remove `../docs/` in the netlify pr (or a follow up)